bower.json

```diff
@@ -13,7 +13,15 @@
     "package.json"
   ],
   "dependencies": {
-    "d2l-colors": "^2.2.3",
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#^1.9.1 || ^2.0.0",
```

Comment:
I see, from [here](https://www.polymer-project.org/2.0/docs/devguide/hybrid-elements#dependency-variants) it has both listed. Interesting.